content/en/platform/corda/4.8/enterprise/notary/faq/notary-load-balancing.md

```diff
@@ -27,28 +27,28 @@ to the appropriate notary worker.
 
 ## Why is the load balancing on the client side?
 
-While server side solutions are possible, client side was chosen for future flexibility. As any notary worker in the
-cluster might be an adversary and deny service, the client needs to have the ability to rotate to different notaries and go
-through each one, until a non faulty one is discovered.
+While server-side solutions are possible, client side was chosen for future flexibility. As any notary worker in the
+cluster might be an adversary, and so deny service, the client needs to have the ability to rotate to different notaries and
+cycle through them until a non-faulty one is discovered.
 
 
 ## What happens if a notary in cluster becomes unavailable and does not respond?
 
-As described in [Notary Failover](notary-failover.md), the client node will retry the flow. That retry goes to Artemis, which because of round
-robin style of communication with the cluster, would send the flow to the next notary worker. For example client sends to notary worker 1, which
-becomes unavailable, client times out while waiting for respond and retries the flow automatically which now gets sent to notary worker 2. Note that
-the retry can also happen in Artemis if notary worker 1 becomes unavailable before acknowledging that it has received the flow message.
+As described in [Notary Failover](notary-failover.md), the client node will retry the flow. That retry goes to Artemis, which because of the round-robin
```

Comment:
```suggestion
As described in [Notary failover](notary-failover.md), the client node will retry the flow. That retry goes to Artemis, which because of the round-robin
```